### DISCUSSION TITLE
implementing Vec postgres fields

### DISCUSSION THREAD

**Discussion by meowser** [SeaQL/sea-orm#1170] [General]
I have a Model implementing a `Vec<i32>` for a postgres int array field like the following:

`#[derive(Clone, Debug, PartialEq, DeriveEntityModel)]
#[sea_orm(schema_name = "some_schema", table_name = "some_table")]
pub struct Model {
    some_field: Option<Vec<i32>>
}`

when I try to compile my program I run into the following error:

| #[derive(Clone, Debug, PartialEq, DeriveEntityModel)]
|                                   ^^^^^^^^^^^^^^^^^ the trait `From<Vec<i32>>` is not implemented for `sea_orm::Value`

Is there a way I can add support for this field?

**Comment by meowser** [comment]
I didn't see this when I first searched, but there is an existing pull requesting and discussion on this, so please ignore

https://github.com/SeaQL/sea-orm/discussions/487
https://github.com/SeaQL/sea-query/pull/205

**Comment by billy1624** [comment]
Hey @meowser, please try SeaORM 0.10.1 :D

**Reply by meowser** [reply]
I added the new feature flag and it works great!
